vulcan/lib/client/jsx/components/dashboard/figure.tsx

```diff
@@ -80,7 +86,7 @@ const Figure = ({
   const {canEdit} = useUserHooks();
 
   const workflow = workflows
-    ? workflows.find((w) => w.name == figureSession.workflow_name)
+    ? (workflows.find((w) => w.name == figureSession.workflow_name) || null)
```

Comment:
super-nit: I actually think there is [an existing selector](https://github.com/mountetna/monoetna/blob/master/vulcan/lib/client/jsx/selectors/workflow_selectors.ts#L30) that might work instead of this?